### PR TITLE
Abass/con30 judge review

### PR DESCRIPTION
**Pull request by abassawo**

### CODE REVIEW

File: app/src/main/java/io/intrepid/contest/models/EntryBallot.java [hunk 1]
```diff
@@ -33,12 +33,4 @@ public boolean isCompletelyScored() {
         }
         return true;
     }
-
-    public float getScoreAverage() {
```
Comment (jmccormack200) [inline app/src/main/java/io/intrepid/contest/models/EntryBallot.java]: Are we not using this anymore?

File: app/src/main/java/io/intrepid/contest/screens/contestjudging/scoreentries/ScoreEntriesActivity.java [hunk 2]
```diff
@@ -104,6 +104,11 @@ public void setNextEnabled(boolean enabled) {
     }
 
     @Override
+    public void returnToEntriesListPage() {
+        showEntriesList();
```
Comment (jmccormack200) [inline app/src/main/java/io/intrepid/contest/screens/contestjudging/scoreentries/ScoreEntriesActivity.java]: show entries list is already public. We can just call it directly.

File: app/src/main/java/io/intrepid/contest/models/EntryBallot.java (hunk 1 above)
Comment (abassawo) [inline app/src/main/java/io/intrepid/contest/models/EntryBallot.java]: Yea this isn't being used anymore now that we are calculating the score within the Entry objects now

Comment (jmccormack200) [inline app/src/main/java/io/intrepid/contest/models/EntryBallot.java]: 👍 gotcha that makes more sense.

**Pull request merged**